src/VirtualClient/VirtualClient.Main/profiles/PERF-MEM-STRESSAPPTEST.json

```diff
@@ -0,0 +1,31 @@
+{
+    "Description": "StressAppTest Workload",
+    "Parameters": {
+        "TimeInSeconds": "60",
+        "UseCpuStressfulMemoryCopy": false
+    },
+    "Actions": [
+        {
+            "Type": "StressAppTestExecutor",
+            "Parameters": {
+                "Scenario": "ApplyStress",
+                "CommandLine": "",
+                "TimeInSeconds": "$.Parameters.TimeInSeconds",
+                "UseCpuStressfulMemoryCopy": "$.Parameters.UseCpuStressfulMemoryCopy",
+                "PackageName": "stressapptest"
+            }
+        }     
+    ],
+    "Dependencies": [
+        {
+            "Type": "DependencyPackageInstallation",
+            "Parameters": {
+                "Scenario": "InstallStressAppTestPackage",
+                "BlobContainer": "packages",
+                "BlobName": "stressapptest_20221115.zip",
```

Comment:
Please also upload to https://virtualclient.blob.core.windows.net/packages, including the proper license file
Make sure you add the .vcpkg file to the source as well.

Reply:
Uploaded the workload package to mentioned Blob storage and added the vcpkg file.